media/json-jackson/src/main/java/org/glassfish/jersey/jackson/internal/DefaultJacksonJaxbJsonProvider.java

```diff
@@ -99,7 +100,14 @@ private List<Module> filterModules() {
                         commonConfig.getRuntimeType(),
                         CommonProperties.JSON_JACKSON_ENABLED_MODULES, String.class);
 
-        final List<Module> modules = ObjectMapper.findModules();
+        final List<Module> modules;
+        try {
+            modules = ObjectMapper.findModules();
+        } catch (Exception e) {
```

Comment:
Just I noted that ObjectMapper will throw an java.util.ServiceConfigurationError. This needs to catch Throwable.